bootstrap.php

```diff
@@ -648,6 +649,9 @@ function ($app) {
         $commandBus->subscribe($app[ProductionCommandHandler::class]);
         $commandBus->subscribe($app[UpdateStatusHandler::class]);
         $commandBus->subscribe($app[ChangeOwnerHandler::class]);
+        $commandBus->subscribe($app[AddLabelHandler::class]);
```

Comment:
I remember that the order of subscribing could have some unwanted side effects. Is that order also important for labels? Or was that more an issue with the event listeners and not the command handlers?

Reply:
You remembered correctly, because you documented that yourself here 😉 https://github.com/cultuurnet/udb3-silex/blob/master/bootstrap.php#L593-L595

It's not clear to me why that is important though, since they listen to different commands. The label command handler handles commands from the `CultuurNet\UDB3\Label` namespace, and the new ones (and the older offer ones) handle commands from the event/place/offer namespace. So the order should not matter.

Reply:
In any case, I tested it manually and I can add unexisting labels without issue

Reply:
I'm going to check if I can still add a new label to an event/place if I move the command handler you put at the top at the end 🤔

Reply:
See 81125c0 (and the commit message for more info)